### PR TITLE
[OSDEV-1387] [hotfix 1.23.1] Improve tile generation performance by eliminating `group by` clause

### PR DESCRIPTION
**Pull request by killev**
Reimplemented tiles generation with PostgreSQL+PostGIS

- avoid using the JOIN + GROUP BY clause. This change reduces the number of subqueries and their asymptotic complexity.
- Enhanced error handling for empty results in facility grid vector tile generation.
- Introduced a constant HI_LIMIT for improved query logic.
- Removed throttling from the get_tile function, allowing unrestricted access.

### CODE REVIEW

File: doc/release/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,25 @@ All notable changes to this project will be documented in this file.
...
+            - In the IAM console, find and click on the "Users" tab.
+            - In the list of available users, locate the `terraform_ci` user, click on it, and on that page, you will find its ARN.
+        - After copying this value, go to the AWS OpenSearch console in the same way you accessed the IAM console.
+        - Open the available domains and locate the domain for the corresponding environment. Open it, then navigate to the security configuration and click "Edit".
+        - Find the section titled "Fine-grained access control", and under this section, you will find an "IAM ARN" input field. Paste the copied ARN into this field and save the changes. It may take several minutes to apply. Make sure that the "Configuration change status" field has green status.
+    3. Then, return to the running `Deploy to AWS` workflow and ensure that the logs for `clear_opensearch` job do not contain errors related to access for deleting the OpenSearch index or lock files in EFS storage. In case of **an access error**, simply rerun the `Deploy to AWS` workflow manually from the appropriate release Git tag.
+
```
Comment (vladsha-dev) [inline doc/release/RELEASE-NOTES.md:24]: It should not be done.

**Review by VadimKovalenkoSNF — APPROVED** [review]
Approved.

**Review by vladsha-dev — CHANGES_REQUESTED** [review]

**Review by roman-stolar — APPROVED** [review]
LGTM

File: doc/release/RELEASE-NOTES.md (hunk 1 above)
Comment (killev) [reply doc/release/RELEASE-NOTES.md:24]: Thank you for pointing that, I copied from 1.23.0, I'm not aware about those rituals. Should I remove?

Comment (vladsha-dev) [reply doc/release/RELEASE-NOTES.md:24]: Yes, it’s not necessary at the moment, as these changes will be required for version 1.24

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Pull request merged**